Content.Server/GameObjects/Components/Doors/AirlockComponent.cs

```diff
@@ -200,27 +201,28 @@ private bool IsPowered()
 
         public bool AttackBy(AttackByEventArgs eventArgs)
         {
-            if (eventArgs.AttackWith.HasComponent<CrowbarComponent>())
-            {
-                if (IsPowered())
-                {
-                    var notify = IoCManager.Resolve<IServerNotifyManager>();
-                    notify.PopupMessage(Owner, eventArgs.User, "The powered motors block your efforts!");
-                    return true;
-                }
+            if (!eventArgs.AttackWith.TryGetComponent<ToolComponent>(out var tool))
+                return false;
 
-                if (State == DoorState.Closed)
-                {
-                    Open();
-                }
-                else if(State == DoorState.Open)
-                {
-                    Close();
-                }
+            if (tool.Behavior != Tool.Crowbar) return false;
```

Comment:
I see you are alternating one liners and two liners `if`s and `[ViewVariable] public bool MyVar;`
I find this confusing.

Reply:
Alright, I cleaned up the code a bit to address most of this. If you find any more examples of this, leave a review.